wallet/src/account/output_cache/mod.rs

```diff
@@ -680,56 +684,92 @@ impl OutputCache {
         ))
     }
 
-    pub fn check_conflicting(&mut self, tx: &WalletTx, block_id: Id<GenBlock>) -> Vec<&WalletTx> {
-        let is_unconfirmed = match tx.state() {
-            TxState::Inactive(_)
-            | TxState::InMempool(_)
-            | TxState::Conflicted(_)
-            | TxState::Abandoned => true,
-            TxState::Confirmed(_, _, _) => false,
-        };
-
-        if is_unconfirmed {
-            return vec![];
+    pub fn update_conflicting_txs(
+        &mut self,
+        confirmed_tx: &Transaction,
+        block_id: Id<GenBlock>,
+    ) -> WalletResult<Vec<Id<Transaction>>> {
+        struct Conflict {
+            frozen_token_id: Option<TokenId>,
+            confirmed_account_nonce: Option<(AccountType, AccountNonce)>,
         }
 
-        let frozen_token_id = tx.inputs().iter().find_map(|inp| match inp {
-            TxInput::Utxo(_) | TxInput::Account(_) => None,
-            TxInput::AccountCommand(_, cmd) => match cmd {
-                AccountCommand::MintTokens(_, _)
-                | AccountCommand::UnmintTokens(_)
-                | AccountCommand::LockTokenSupply(_)
-                | AccountCommand::ChangeTokenMetadataUri(_, _)
-                | AccountCommand::ChangeTokenAuthority(_, _)
-                | AccountCommand::UnfreezeToken(_)
-                | AccountCommand::ConcludeOrder(_)
-                | AccountCommand::FillOrder(_, _, _) => None,
-                AccountCommand::FreezeToken(frozen_token_id, _) => Some(frozen_token_id),
-            },
+        let conflict = confirmed_tx.inputs().iter().find_map(|input| {
+            match input {
+                TxInput::Utxo(_) => {
+                    //TODO: check conflicting utxo spends
```

Comment:
Is there any reason for not implementing it right now?

Reply:
Just wanted to focus on accounts because it already took me longer than I expected